ios/FluentUI/AvatarGroup/AvatarGroupTokens.swift

```diff
@@ -6,87 +6,95 @@
 import UIKit
 import SwiftUI
 
-class MSFAvatarGroupTokens: MSFTokensBase, ObservableObject {
-    @Published public var interspace: CGFloat!
-    @Published public var ringInnerGap: CGFloat!
-    @Published public var ringThickness: CGFloat!
-    @Published public var ringOuterGap: CGFloat!
-
-    var style: MSFAvatarGroupStyle {
-        didSet {
-            if oldValue != style {
-                updateForCurrentTheme()
-            }
-        }
-    }
-
-    var size: MSFAvatarSize {
-        didSet {
-            if oldValue != size {
-                updateForCurrentTheme()
-            }
-        }
-    }
-
-    init(style: MSFAvatarGroupStyle,
-         size: MSFAvatarSize) {
+/// Design token set for the `AvatarGroup` control
+public class AvatarGroupTokens: ControlTokens {
+    /// Creates an instance of `AvatarGroupTokens`with optional token value overrides.
+    /// - Parameters:
+    ///   - interspace: CGFloat that defines the space between  the `Avatar` controls hosted by the `AvatarGroup`
+    ///   - ringInnerGap: CGFloat that defines the thickness of the space between the `Avatar` and its ring
+    ///   - ringThickness: CGFloat that defines the thickness of the ring around the `Avatar`
+    ///   - ringOuterGap: CGFloat that defines the thickness of the space around the ring of the `Avatar`
+    public init (style: MSFAvatarGroupStyle,
+                 size: MSFAvatarSize,
+                 interspace: CGFloat? = nil,
+                 ringInnerGap: CGFloat? = nil,
+                 ringThickness: CGFloat? = nil,
+                 ringOuterGap: CGFloat? = nil) {
         self.style = style
         self.size = size
 
         super.init()
 
-        self.themeAware = true
+        // Optional overrides
+        if let interspace = interspace {
+            self.interspace = interspace
+        }
 
-        updateForCurrentTheme()
-    }
+        if let ringInnerGap = ringInnerGap {
+            self.ringInnerGap = ringInnerGap
+        }
+
+        if let ringThickness = ringThickness {
+            self.ringThickness = ringThickness
+        }
 
-    @objc open func didChangeAppearanceProxy() {
-        updateForCurrentTheme()
+        if let ringOuterGap = ringOuterGap {
+            self.ringOuterGap = ringOuterGap
+        }
     }
 
-    override func updateForCurrentTheme() {
-        let currentTheme = theme
-        var appearanceProxy: AppearanceProxyType
-        let avatarTokens = currentTheme.MSFAvatarTokens
+    // MARK: - Design Tokens
+
+    let style: MSFAvatarGroupStyle
+    let size: MSFAvatarSize
 
+    lazy var interspace: CGFloat = {
         switch style {
         case .stack:
-            appearanceProxy = currentTheme.MSFAvatarGroupStackTokens
+            switch size {
+            case .xsmall, .small:
+                return -2 // -spacing[.xxxsmall]
```

Comment:
Why are these values constants instead of references to the relevant global tokens?

Reply:
They were constants in the old tokens so I left them as is. I wasn't sure if there was a reason we didn't just do the negative spacing values I commented next to them, though.